Project.xml

```diff
@@ -2,7 +2,7 @@
 <project>
 	<!-- _________________________ Application Settings _________________________ -->
 
-	<app title="Friday Night Funkin': Psych Engine" file="PsychEngine" packageName="com.shadowmario.psychengine" package="com.shadowmario.psychengine" main="Main" version="0.2.7" company="ShadowMario" />
+	<app title="Friday Night Funkin': Psych Engine" file="PsychEngineButBetter" packageName="com.shadowmario.psychengine" package="com.shadowmario.psychengine" main="Main" version="0.2.7" company="ShadowMario" />
```

Comment:
Yeah, don't change it. It will affect saves and data caching.

Reply:
>Ruins all data saves

Reply:
-- not even gonna go into how, uhhh, "less than a good idea" changing the name of the actual engine to where it seems like it's a mod of said engine is